src/platform/site-wide/user-nav/tests/containers/AutoSSO.unit.spec.jsx

```diff
@@ -20,51 +20,6 @@ describe('<AutoSSO>', () => {
     };
   });
 
-  it('should not call setForceAuth if auth query param is set to "success"', () => {
```

Comment:
Should the `!useInboundSSOe` bit be tested instead?

Reply:
yeah, I'll add a couple of tests